android/app/build.gradle

```diff
@@ -111,6 +111,8 @@ android {
         versionName "1.6.0"
         vectorDrawables.useSupportLibrary = true
         missingDimensionStrategy 'react-native-camera', 'general'
+        testBuildType System.getProperty('testBuildType', 'debug')  // This will later be used to control the test apk build type
+        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
```

Comment:
Can we add `androidx` dependencies to our RN59 app?

Reply:
I was surprised by this line as well and intended to investigate it like a week ago, but then totally forgot because, well, it works...... so I guess, to answer your question: yes, looks like it

Reply:
No actually I take that back. I think this is only for the "instrumentation runner". Detox installs another app separately for running instrumentation. I don't think we can mix in androidx stuff into our own app. (haven't confirmed 100%)